_src/css/helpcenter/_fonts.scss

```diff
@@ -20,6 +20,12 @@ body {
   font-family: Graphik-Regular, sans-serif;
 }
 
+// bold text doesnt stand out in graphik regular
+body strong {
+  font-family: Graphik-Medium, sans-serif;
+  color: #000;
```

Comment:
Are we deliberately overriding the color to a slightly blacker black than the rest of the text?

Reply:
Yeah, it stands out better as full black than the dark grey